src/ResourceManager/ManagementGroups/Commands.ManagementGroups/Cmdlets/NewAzureRmManagementGroup.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ManagementGroups.Common;
+using Microsoft.Azure.Commands.ManagementGroups.Models;
+using Microsoft.Azure.Management.ManagementGroups;
+using Microsoft.Azure.Management.ManagementGroups.Models;
+
+namespace Microsoft.Azure.Commands.ManagementGroups.Cmdlets
+{
+    /// <summary>
+    /// New-AzureRmManagementGroup Cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "AzureRmManagementGroup", DefaultParameterSetName = Constants.ParameterSetNames.GroupOperationsParameterSet, SupportsShouldProcess = true), OutputType(typeof(PSManagementGroupNoChildren))]
+    public class NewAzureRmManagementGroup : AzureManagementGroupsCmdletBase
+    {
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.GroupOperationsParameterSet, Mandatory = true, HelpMessage = Constants.HelpMessages.GroupName, Position = 0)]
+        [ValidateNotNullOrEmpty]
+        public string GroupName { get; set; }
+
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.GroupOperationsParameterSet, Mandatory = false,
+            HelpMessage = Constants.HelpMessages.DisplayName)]
+        [ValidateNotNullOrEmpty]
+        public string DisplayName { get; set; } = null;
+
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.GroupOperationsParameterSet, Mandatory = false,
+            HelpMessage = Constants.HelpMessages.ParentId)]
+        [ValidateNotNullOrEmpty]
+        public string ParentId { get; set; } = null;
```

Comment:
@rajshah11 do you think it would make sense to have another parameter set that allows the user to pass in the `PSManagementGroup` object of the parent that they want for this new management group?

Reply:
Yes. I think it would make sense.